source/development/notifier.rst

```diff
@@ -9,8 +9,7 @@ To run tests, first get all dependencies.
 
 .. code-block:: bash
 
-   go get github.com/kardianos/govendor
-   govendor sync
+   go get ./...
```

Comment:
Do we need this at all? Maybe `goconvey` runs without explicitly `go get`-ting dependencies? Please, investigate.

Reply:
Fixed